### DISCUSSION TITLE
Should I create SSHOptions to call connection_sharing_args?

### DISCUSSION THREAD

**Discussion by groves** [kovidgoyal/kitty#5177] [Q&A] [answered]
[kittens/ssh/main.py:connection_sharing_args](https://github.com/kovidgoyal/kitty/blob/9f5f113840be16a709fddd294a079eb714a537cd/kittens/ssh/main.py#L511) takes an SSHOptions instance but doesn't use them. I'm currently passing `None` for the options and it's working fine. Should I create options using some config parsing as future proofing, or is `None` fine?

**Comment by kovidgoyal** [comment, marked as answer]
It's not needed. I have removed it. If in the future there is something else that can influence the path, then it can be passed in individually rather than using the full opts object.
